src/test/java/teammates/test/BaseTestCaseWithLocalSqlDatabaseAccess.java

```diff
@@ -23,6 +24,9 @@ public static void startContainer() {
         FlywayUtil.getFlywayInst(pgsql.getJdbcUrl(), pgsql.getUsername(), pgsql.getPassword()).migrate();
         HibernateUtil.setSessionFactoryForTesting(
                 pgsql.getUsername(), pgsql.getPassword(), pgsql.getJdbcUrl());
+
+        Session currentSession = HibernateUtil.getSessionFactory().getCurrentSession();
+        currentSession.beginTransaction();
```

Comment:
I think we also need to close the session in the `closeContainer` method.

Reply:
added!